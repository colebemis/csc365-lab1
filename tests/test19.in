CLASSROOM: 112 TEACHER
Q