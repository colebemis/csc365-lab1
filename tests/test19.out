Enter query: CHIONCHIO, PERLA
Enter query: 